### DISCUSSION TITLE
Keyboard multiple response alternative

### DISCUSSION THREAD

**Discussion by dusannikolic92** [jspsych/jsPsych#2227] [Q&A] [answered]
Hello all,

I am trying to have my participants listen to the sequence of words and then, only after the sequence has finished, press multiple keys (e.g. A and L, so that the answer is for instance ALLAL). I have been trying to do this with the ```html-keyboard-multiple-response``` but there are two issues/questions: 

1. Is it possible to do this with a newer plugin? This one is supported on jspsych 6.0.5, but I am not sure about later versions, and the coding has changed since so my code is different for the newer versions of jspsych.
2. How can I use this plugin and have the answers after the stimuli have finished playing? The ```response_allowed_while_playing: false``` is not working with multiple-response plugin.

Thank you so much in advance. Any help is appreciated !

Best,
Dusan

**Comment by jodeleeuw** [comment, marked as answer]
Hi @dusannikolic92,

Is the length of the response fixed for each sequence? If so, you could chain together multiple trials of the `html-keyboard-response` plugin. This was actually why we removed the multiple response version, because in most cases it works to just use the single response version, so it was easier to maintain the simpler version.

**Reply by dusannikolic92** [reply]
Hello @jodeleeuw,

There is no time limit. When participants put in their response, they press Enter, and the trial is completed. 
I have stopped trying to use this plugin, and I will be moving on to 'survey-text' plugin which enables participants to insert their response into the textbox instead. It is much simpler.

However, I might ask about the play button and how I can have a play button displayed so that participants can play the stimuli as many times as they want. If you have any guidelines on that, I would sincerely appreciate it.

Thank you!
Dusan
